### PR TITLE
Ignores Biography check if deleted

### LINKED ISSUE #132: Security Analyzer : Biography field is deleted but still shows Rich Text warning
Moved from https://github.com/DNNCommunity/DNN.SecurityAnalyzer/issues/25

Originally opened by @kingjordan

I have a DNN 8 website where I have deleted the biography from the user profile items. The warning for the biography field being Rich text is still showing

### PR DESCRIPTION
**Pull request by valadas**
Closes #132 

## Summary
In the security analyzer, no longer reports an error on the Briography profile property being richtext if that profile property is deleted.

### CODE REVIEW

File: Extensions/Settings/Dnn.PersonaBar.Security/admin/personaBar/App_LocalResources/Security.resx [hunk 1]
```diff
@@ -820,7 +820,7 @@
     <value>The biography field is a common target for spammers as they can add links/html to it. In DNN 7.2.0 this was changed to a multiline textbox which removes this risk.</value>
   </data>
   <data name="CheckBiographySuccess.Text" xml:space="preserve">
-    <value>The field is a multiline textbox</value>
+    <value>The field not enabled or is a multiline textbox</value>
```
Comment (daguiler) [inline Extensions/Settings/Dnn.PersonaBar.Security/admin/personaBar/App_LocalResources/Security.resx]:
```suggestion
    <value>The field is not enabled or is a multiline textbox</value>
```

**Review by daguiler — COMMENTED** [review]
Added a suggestion

**Comment by valadas** [comment]
> Added a suggestion

Good catch, thanks.

**Review by daguiler — APPROVED** [review]

**Review by david-poindexter — APPROVED** [review]
LGTM - thanks @valadas

**Pull request merged**